img2dataset/downloader.py

```diff
@@ -25,9 +46,13 @@ def download_image(row, timeout):
         request = urllib.request.Request(
             url,
             data=None,
-            headers={"User-Agent": "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:72.0) Gecko/20100101 Firefox/72.0"},
+            headers={
+                "User-Agent": "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:72.0) Gecko/20100101 Firefox/72.0 (compatible; img2dataset; +https://github.com/rom1504/img2dataset)"
```

Comment:
if you want you could add an option for this and leave the current thing as default